doc/interop-test-descriptions.md

```diff
@@ -1143,10 +1147,14 @@ Procedures:
       }
     }
     ```
-The call carries a reference to receive the load report, e.g. using CallOptions.
-The reference will be passed to the custom LB policy as part of the `OrcaOobReportListener` API.
-* Client asserts that, after 1.5 second, the latest OOB load report received is equal to the test load report.
-* Client sends another unary call to the server. The call request sets `orca_oob_report` to a 
+The test then blocks until the first response is received. This means the server lock is acquired.
```

Comment:
Take a look at ping_pong for how simply we try to express the test process. Terseness is helpful because it makes it much easier to verify a test implementation is correct.

This can just be a step (2) "Client waits for a reply." Stuff like "this means" should be removed. The "it is important" was important to the Java implementation, and all implementations should consider stuff like that, but isn't a good fit for discussing here. If anything, just say "Client asserts it receives a reply."

Yes, we were careful on the implementation side of things, but that was really about me hating bad error codes. How that is done in each language and what turns out to be important in each language will vary.